src/test/java/seedu/address/storage/JsonAdaptedCompanyTest.java

```diff
@@ -28,7 +26,10 @@ public class JsonAdaptedCompanyTest {
     private static final String INVALID_URL = " ";
     private static final String INVALID_EMAIL = "example.com";
     private static final String INVALID_TAG = "#friend";
-    private static final String INVALID_REMARK = "   "; // Assuming a completely blank remark is invalid.
+    private static final String INVALID_REMARK = "q8ElAFCNZ5)-:Q7$-}m4KH#1QF*HI-tiR,1o"
+            + "`Hg}1K86Lm`t-0$oS)hY,31+*`t#^W|?tm3%2-px#[,/>Mqrgcvam~Kq7|.EN6,7a8jxI1mZw]p_n|0<DA=Rl)[rwh3+gyL$S')"
+            + "dtBasijifhihsfihoajod9138943409734102914/1afC60L~5(^Ph?eO9Q-mP,</^OL&+q}oWnkIY)*Uv_-aUmYfe^"
+            + ";({K1B";
```

Comment:
I see that you changed the invalid remark so that a blank remark is not invalid and the test cases can work